server/autolinkplugin/command.go

```diff
@@ -80,6 +84,42 @@ func (ch CommandHandler) Handle(p *Plugin, c *plugin.Context, header *model.Comm
 	return ch.defaultHandler(p, c, header, args...)
 }
 
+func executeExperimental(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+	if len(args) == 0 {
+		return responsef("Please select a subcommand")
+	}
+
+	if args[0] != "app" {
+		return responsef("Unknown subcommand")
+	}
+
+	if len(args) == 1 {
+		return responsef("Please select a setting")
+	}
+
+	if args[1] == "off" {
+		return responsef("App is already disabled")
+	}
+
+	if args[1] != "on" {
+		return responsef("Please select either `off` or `on`")
+	}
+
+	appsPluginClient := mmclient.NewAppsPluginAPIClientFromPluginAPI(&pluginapi.NewClient(p.API, p.Driver).Plugin)
+
+	err := appsPluginClient.InstallApp(autolinkapp.Manifest)
+	if err != nil {
+		return responsef("error occurred while installting the autolink app: %v", err)
+	}
+
+	apiError := p.API.UnregisterCommand("", "autolink")
```

Comment:
Should we get the command trigger in a constant?

Reply:
Good suggestion. Added one in cf41432.